recipes/mbedtls/all/conanfile.py

```diff
@@ -90,9 +97,25 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "cmake"))
 
     def package_info(self):
-        # https://gitlab.kitware.com/cmake/cmake/blob/de7c21d677db1ddaeece03c19e13e448f4031511/CMakeLists.txt#L380
         self.cpp_info.names["cmake_find_package"] = "MbedTLS"
         self.cpp_info.names["cmake_find_package_multi"] = "MbedTLS"
-        self.cpp_info.libs = ["mbedtls", "mbedx509", "mbedcrypto"]
+        self.cpp_info.components["mbedcrypto"].names["cmake_find_package"] = "mbedcrypto"
```

Comment:
Components are only defined on version 3.0.0.
I guess we want to expose them anyways for 2.x.
If that's not the case I'll wrap this section on a version check.

Reply:
I think it's okay, the global `MbedTLS::MbedTLS` target is still present, so it's completely backwards compatible.

Reply:
My thoughts exactly.